Ollivanders/src/net/pottercraft/ollivanders2/Ollivanders2Common.java

```diff
@@ -88,6 +95,11 @@
       add(EntityType.POLAR_BEAR);
       add(EntityType.LLAMA);
       add(EntityType.SHULKER); // not large in size but complex
+      add(EntityType.PANDA);
+      add(EntityType.DOLPHIN);
+      add(EntityType.SNOWMAN);
+      add(EntityType.SKELETON_HORSE);
```

Comment:
Is it intentional to have skeleton horse as large friendly and undead

Reply:
This would be used for any spells that only work on undead, which they are. I guess they are not strictly speaking "friendly" so I removed them from friendly.